tests/installation/releasenotes_online.pm

```diff
@@ -0,0 +1,37 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017-2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Check origin of displayed releasenotes during installation
+# Maintainer: mgriessmeier <mgriessmeier@suse.com>, Nick Singer <nsinger@suse.de>
+# Tags: fate#323273, poo#26786, #poo97316
+
+use base 'y2_installbase';
+use strict;
+use warnings;
+use testapi;
+use version_utils ':VERSION';
```

Comment:
you can remove lines 32-34 because we don't need to check for the next screen, next test just does that, so we can also remove this line importing the function.